content/logs/index.md

```diff
@@ -107,6 +107,32 @@ logs:
 
 {{< /highlight >}}
 
+### Docker log collection
+
+Agent 6 is able to collect logs from containers. It can be installed [on the host](https://github.com/DataDog/datadog-agent/blob/master/docs/beta/upgrade.md) or [in a container](https://github.com/DataDog/datadog-agent/tree/master/Dockerfiles/agent).
+
+To start collecting logs for a given container filtered by image or label, you need to update the log section in an integration or custom yaml file.
```

Comment:
- you need to update the log section in an integration or custom yaml file.

+ Update the integration log section in its yaml file, or create a [custom yaml file](/logs/#custom-log-collection)

Reply:
👍